packages/quantic/force-app/main/default/lwc/quanticRecommendationList/e2e/quanticRecommendationList.e2e.ts

```diff
@@ -0,0 +1,53 @@
+import {test, expect} from './fixture';
+
+const testFieldsToInclude = 'foo,bar,baz';
+
+test.describe('quantic recommendation list', () => {
+  test.describe('when clicking a recommendation', () => {
+    test('should log recommendation open analytics', async ({
+      recommendationList,
+    }) => {
+      await recommendationList.captureRecommendationListClickEventWorkaround();
+
+      const recommendationClickAnalyticsPromise =
+        recommendationList.waitForRecommendationListClickEvent(
+          'recommendationOpen'
+        );
+
+      const firstRecommendationLink =
+        recommendationList.getRecommendationLinkByIndex(0);
+      firstRecommendationLink.scrollIntoViewIfNeeded();
+      await firstRecommendationLink.evaluate((el) =>
+        el.setAttribute('href', '#')
+      );
+      await firstRecommendationLink.click();
+
+      await recommendationClickAnalyticsPromise;
+    });
+  });
+
+  test.describe('fieldsToInclude in the Search API request', () => {
+    test.use({
+      options: {
+        fieldsToInclude: testFieldsToInclude,
+      },
+    });
+
+    test('when performing a search should send the added fieldsToInclude', async ({
+      recommendationList,
+      search,
+    }) => {
+      const expectedFieldsToInclude = testFieldsToInclude.split(',');
+      const searchRequestPromise = search.waitForSearchRequest();
+      await search.performRecommendationSearch();
+      const requestBody = (await searchRequestPromise).postDataJSON();
+      expect(requestBody?.fieldsToInclude).toBeDefined();
+      expect(requestBody).toEqual(
+        expect.objectContaining({
+          fieldsToInclude: expect.arrayContaining(expectedFieldsToInclude),
+        })
+      );
```

Comment:
There is a special field in the search request that specifies the search is for recommendation purposes

It's something like `isRecommendation: true|false`, you should test for that as well.

Reply:
you mean recommendation: "Recommendation"? 

<img width="526" alt="image" src="https://github.com/user-attachments/assets/7b9702a5-3e26-4eef-89a4-f621045f8c64" />

Reply:
added a check for the recommendation